tensorflow/tools/ci_build/install/install_golang_ppc64el.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
Minor nit.
filename, should it be *ppc64le.sh

Reply:
Thanks for the comment, I fixed the filename and it's invocation from the two dockerfiles.